devops/dev/templates/jwt-keys-sealed-secret.yaml

```diff
@@ -6,11 +6,12 @@ metadata:
+    public.pem: AgAXq8k2/zdo1jyVp1LD9wQ+mQ11O+ocxhG+WCeA981LQTQVvqdsiNAQjoFwIh7I4LVgoo7SLRgkkpX82EDBzWihMI8yJkyZINE8rRp5sK3Ka3mpXnFoFaRrIplse4v3RdrZY+FByWRvIBWq0zdzP6PyfNgSEdXLmVkvZl3XfUWJglUnPiKsdS5p3MXRd0EWT3e6poTVKIX7LqU5ih1tnIKg62rDHTdHRVzNuKuqcH6zM5JwK/924IufuxOUBzzO40NeXMrzGkl06zRydDzxd7cB/jXfUc3wb3l35AqY25rvZ0R3UouL9TKYXTobis+mLH+NcZXGdYVGv2IOK1wfNY1qfbGjRyeTfxvjlAg9HZ0YqSQkR3l+uZRg93Gs9ht+cw2Rr0OgDrCAVBYUwRl5DGKca78YuhE45YWdTBOqv6uHStwM8BfYShJY68w42WdBv4T+zpSEdZLeuz8vHHEQ8stQhdJ8f8Cym2UjmXVCsoap3UXHxwNYuGyD5DMGChvdf4FxgoGEewCaYn7AeRVK4OUuMcZ7AeFzfy1Evjj2QE90dSTnMHg6SV6AojVmbrWIK1nhhlDwFCxCTIlSx6JuJxqQsxnWw+aEvWpEytPdaokdBc5HhVlnA4CPqY+y5z/7welaFTlaY8S/b3lNDXC7CUOvdU9jcz84L7nT81wWcBmaedTZxK8xUvGI+xX2J2hNHhDP1jfe4fY2ibJZs8uODVA2N8wCPQXoTJM8yPGvdAej996udaTTKolFwdfL9q/0Nv+Gj7e/JLVWgiqvTOAdbkkfPZlTcVVq2AfrdB6iCfDG6hX6geq++NGuWDliJ8IOq4t1gMi25gPN7Wn4viqBybnyaKicNe9/y3moGdPgUJRqC8a13uzqwIKPLoaQPbhQtddK9TJauABFOhgAqoKSHL0YmjCxpwxWpafidOp7LO7v346YvMaCRWyvf5L05s5m6soXOMyh2CVkur11efH50cITojuA5mlbckApTw2Xq3gvtFxuPJvB183Q3ltJ9BowP/fWSyx1DK1lyPHCLr3Ggpx30R2KJ1oBIEkmMeFfvsAVDaCn4t4n+u5XbW7AgnFt1ii5/b41FJBF3mDK62g5nVsVM3/ocGU+lZXsBrmwcPBLB5jeCn4iKwKxFIjfWO76Jn8RrrRa5Hy3IW6HaaTSJXpXoRUhALxXut6F8OevfUsOXWhu8cvU9FQkRl/E2cd5fDdhTj7WW31H4VZ/sL4m5BE0X4SanqHJdUYR2l45imh6amLqV+5Smq8M8BGehcvEsdQEbm7CiUpXwh7bkcBnCue8/jnh
   template:
     metadata:
       creationTimestamp: null
       name: jwt-keys
       namespace: ics
+    type: Opaque
```

Comment:
@michalziobro what does this `Opaque` do?

Reply:
@jrhender there are various types of secrets in k8s, Opaque is let's say a 'standard' secret:
https://stackoverflow.com/questions/49614439/kubernetes-secret-types